src/storybook-components/Page.vue

```diff
@@ -99,6 +99,9 @@
       </template>
 
       <template slot="belowContent">
+        <div class="rpl-site-constrain--on-all">
+          <rpl-card-carousel :title="mock.cardCarousel.title" :cards="mock.cardCarousel.cards" />
```

Comment:
Hi @alan-cole can you please place this component in content slot? it suppose to be a landing page body component.

Reply:
Updated - please note that it is now affected by the sidebar. Sidebar should be disabled to correctly view.